vagrant/install_mhVTL.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+echo "Script Begin"
+
+get_os_name(){
+    if [[ "$(hostnamectl | grep -i ubuntu | wc -l)" != "0" ]]; then
+        OS_NAME='ubuntu'
+    elif [[ "$(hostnamectl | grep -i sles | wc -l)" != "0" ]]; then
+        OS_NAME='sles'
+    elif [[ "$(hostnamectl | grep -i opensuse | wc -l)" != "0" ]]; then
+	OS_NAME='opensuse'
```

Comment:
```suggestion
        OS_NAME='opensuse'
```

Reply:
I wonder how this happened...great catch! :)